src/termmouse.c

```diff
@@ -40,29 +92,10 @@ static char scheme = 0;
   for (;;) {
     wchar c = get_char(line, p.x);
 
-    // scheme detection state machine
     if (!forward) {
-      // http://abc.xy
-      //0ssss://
-      if (isalnum(c)) {
-        if (scheme == ':')
-          scheme = 's';
-        else if (scheme != 's')
-          scheme = 0;
-      }
-      else if (c == ':') {
-        if (scheme == '/')
-          scheme = ':';
-        else
-          scheme = 0;
-      }
-      else if (c == '/') {
-        scheme = '/';
-      }
-      else if (scheme)
```

Comment:
If not wanting to use the "proper" state machine in my PR, the "correct" fix on line 62 would probably be:

```c
else if (scheme == 's')
```
and probably change line 50-51 to:
```c
else
  scheme = 0;
```

That way the selection loop stops only if the state has been elevated to scheme.